pages/[username].js

```diff
@@ -86,7 +86,7 @@ export default function User({ data, BASE_URL }) {
         target="_blank"
         className="fixed bottom-5 right-5 rounded-full focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-secondary-high"
       >
-        <div className="px-4 py-2 bg-tertiary-medium text-primary-low flex items-center gap-1 rounded-full hover:bg-secondary-medium hover:drop-shadow-lg">
+        <div className="hidden md:flex px-4 py-2 bg-tertiary-medium text-primary-low items-center gap-1 rounded-full hover:bg-secondary-medium hover:drop-shadow-lg">
```

Comment:
The wrapping link (above) will still be in the dom. This should go on there parent tag, I will move in PR https://github.com/EddieHubCommunity/BioDrop/pull/9178